### PR TITLE
home screen design tweaks

### PR DESCRIPTION
**Pull request by michaelkirk-signal**
PTAL @charlesmchen 

screenshots sent OOB

### CODE REVIEW

File: Signal/src/ViewControllers/HomeView/HomeViewCell.m [hunk 1]
```diff
@@ -65,20 +61,19 @@ - (void)commontInit
     OWSAssert(!self.avatarView);
 
     [self setTranslatesAutoresizingMaskIntoConstraints:NO];
-    self.layoutMargins = UIEdgeInsetsZero;
-    self.contentView.layoutMargins = UIEdgeInsetsZero;
-    self.preservesSuperviewLayoutMargins = NO;
-    self.contentView.preservesSuperviewLayoutMargins = NO;
+    self.layoutMargins = UIEdgeInsetsMake(0, self.cellHMargin, 0, self.cellHMargin);
+    self.contentView.preservesSuperviewLayoutMargins = YES;
```
Comment (charlesmchen-signal) [inline Signal/src/ViewControllers/HomeView/HomeViewCell.m]: Don't we still need `self.contentView.layoutMargins = UIEdgeInsetsZero;`?

File: Signal/src/ViewControllers/HomeView/HomeViewCell.m [hunk 2]
```diff
@@ -178,34 +165,71 @@ - (void)configureWithThread:(TSThread *)thread
...
-    if (unreadCount > 0) {
-        [self.topRowView addArrangedSubview:self.unreadBadgeContainer];
+    if (unreadCount == 0) {
+        [self.viewConstraints addObject:[self.payloadView autoPinTrailingToSuperviewMargin]];
+    } else {
+        [self.contentView addSubview:self.unreadBadge];
 
-        self.unreadLabel.font = [UIFont ows_dynamicTypeCaption1Font];
-        self.unreadLabel.text = [OWSFormat formatInt:MIN(99, (int)unreadCount)];
+        self.unreadLabel.text = [OWSFormat formatInt:unreadCount];
```
Comment (charlesmchen-signal) [inline Signal/src/ViewControllers/HomeView/HomeViewCell.m]: We need to update `unreadLabel.font` here or it won't reflect changes to dynamic type size.

File: Signal/src/ViewControllers/HomeView/HomeViewCell.m (hunk 1 above)
Comment (michaelkirk-signal) [reply Signal/src/ViewControllers/HomeView/HomeViewCell.m]: Done!

File: Signal/src/ViewControllers/HomeView/HomeViewCell.m [hunk 3]
```diff
@@ -120,25 +115,17 @@ - (void)commontInit
     [self.snippetLabel setCompressionResistanceHorizontalLow];
 
     self.unreadLabel = [UILabel new];
-    self.unreadLabel.font = [UIFont ows_dynamicTypeCaption1Font];
+    self.unreadLabel.font = self.dateTimeFont;
```
[the remark this answers is not in the document]
Comment (michaelkirk-signal) [reply Signal/src/ViewControllers/HomeView/HomeViewCell.m]: You're right, that's a better name. Updated!

File: Signal/src/ViewControllers/HomeView/HomeViewCell.m (hunk 2 above)
Comment (michaelkirk-signal) [reply Signal/src/ViewControllers/HomeView/HomeViewCell.m]: Good catch, thanks!

**Pull request merged**